README.md

```diff
@@ -8,8 +8,17 @@
 [![Coverage Status](https://coveralls.io/repos/github/mosdef-hub/foyer/badge.svg?branch=master)](https://coveralls.io/github/mosdef-hub/foyer?branch=master)
 [![codecov](https://codecov.io/gh/mosdef-hub/foyer/branch/master/graph/badge.svg)](https://codecov.io/gh/mosdef-hub/foyer)
 
-Annotate an [OpenMM .xml force field](http://docs.openmm.org/7.0.0/userguide/application.html#creating-force-fields)
-file with SMARTS-based atomtypes:
+## Overview
+Foyer is an open-source Python tool for defining and applying force field atom-typing 
+rules in a format that is both human- and machine-readable.  Foyer provides a framework for force field 
```

Comment:
Should we say it only applies the rules or also applies the force field itself? We are working on splitting the two up in a clear manner but right now it does everything at once

Reply:
I think since this is the way it works currently, and the way it is described in the paper, we should leave it as is? Once we make the changes, we can  update this?

Reply:
I think we should say it applies the forcefields and generates syntactically correct input files for various simulation engines.

Reply:
I agree